altanalyze3/components/annotation/ensembl_biomart.py

```diff
@@ -0,0 +1,503 @@
+"""
+This is a generalized python module for getting data from Ensemble using Biomart server.
+"""
+
+from __future__ import absolute_import, division, print_function
+import requests
+
+from future.utils import native_str
+from builtins import *
```

Comment:
import * is usually not recommended in Python

Reply:
Agreed.